lib/hipe/test/basic_SUITE_data/basic_tuples.erl

```diff
@@ -73,6 +75,16 @@ get_elements([Element|Rest], Tuple, Pos) ->
 get_elements([], _Tuple, _Pos) ->
   ok.
 
+squares() ->
+  {1*1,   2*2,   3*3,   4*4,   5*5,   6*6,   7*7,   8*8,   9*9,   10*10,
+   11*11, 12*12, 13*13, 14*14, 15*15, 16*16, 17*17, 18*18, 19*19, 20*20}.
+
+test_square() ->
+  %% A bit of gymnastics to create a variable with a 1..19 range
```

Comment:
The comment is a bit confusing -- the next line creates something in the 10..18 range, or not?

More importantly, I do not understand the test here.  All other tests in `test_element/4` call the test and explicitly match for the expected result.  Besides breaking this pattern, what does this one do?